.github/CODEOWNERS

```diff
@@ -26,17 +26,18 @@
 /lib/asserts.nix            @edolstra @nbp @Profpatsch
 
 # Nixpkgs Internals
-/default.nix                          @nbp
-/pkgs/top-level/default.nix           @nbp @Ericson2314
-/pkgs/top-level/impure.nix            @nbp @Ericson2314
-/pkgs/top-level/stage.nix             @nbp @Ericson2314 @matthewbauer
-/pkgs/top-level/splice.nix            @Ericson2314 @matthewbauer
-/pkgs/top-level/release-cross.nix     @Ericson2314 @matthewbauer
-/pkgs/stdenv/generic                  @Ericson2314 @matthewbauer
-/pkgs/stdenv/cross                    @Ericson2314 @matthewbauer
-/pkgs/build-support/cc-wrapper        @Ericson2314 @orivej
-/pkgs/build-support/bintools-wrapper  @Ericson2314 @orivej
-/pkgs/build-support/setup-hooks       @Ericson2314
+/default.nix                                     @nbp
```

Comment:
I would prefer reverting this to make backports easier.

Reply:
As mentioned in the commit message, I changed this to make it consistent with all the other entries. only adding that one line makes it pretty much stand out.